resources/authgear/templates/en/web/authflowv2/select_account.html

```diff
@@ -14,7 +14,7 @@ <h1 class="screen-title">
       {{ end }}
     </h1>
     <p class="screen-description">
-      {{ template "v2-select-account-description" (dict "IdentityDisplayName" $.IdentityDisplayName) }}
+      {{ template "v2-select-account-description" (dict "UserProfileDisplayName" (include "__user_profile_display_name.html" .)) }}
```

Comment:
```
(merge (dict) .UserProfile (dict "IdentityDisplayName" $.IdentityDisplayName))
```

Define UserProfile as

```
	Email               = "email"
	EmailVerified       = "email_verified"
	PhoneNumber         = "phone_number"
	PhoneNumberVerified = "phone_number_verified"
	PreferredUsername   = "preferred_username"
	FamilyName          = "family_name"
	GivenName           = "given_name"
	MiddleName          = "middle_name"
	Name                = "name"
	Nickname            = "nickname"
	Picture             = "picture"
	Profile             = "profile"
	Website             = "website"
	Gender              = "gender"
	Birthdate           = "birthdate"
	Zoneinfo            = "zoneinfo"
	Locale              = "locale"
```

- If the attribute is a string and the user does not have it, then output "nil" string
- If the attribute is a string and the user has it, then output the value.
- If the attribute is a boolean and the user does not have it, then output "nil" string
- If the attribute is a boolean and the user has it, then output the value.

Reply:
As per discussion, updated to add `xxx_is_present` flag to know a field's presence